src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Tabs/basic.html.twig

```diff
@@ -28,6 +28,29 @@
 <div role="tabpanel" class="form-contenttab tab-pane container-fluid active" id="basic-tab">
   <div class="row">
     <div class="col-md-9 left-column">
+      <div 
+        id="product-images-container"
+        data-translations="{{
+          {
+            'window.selectAll': 'Select all'|trans({}, 'Admin.Actions'),
+            'window.unselectAll': 'Unselect all'|trans({}, 'Admin.Actions'),
+            'window.selectedFiles': '[1]%filesNb%[/1] selected file(s)'|trans({'[1]': '<span>', '[/1]': '</span>'}, 'Admin.Actions'),
```

Comment:
'[1]%filesNb%[/1] selected file(s)'
=> Admin.Catalog.Feature 

@jolelievre Sorry I missed this one.

Reply:
This one has been moved in another file But you already commented it in https://github.com/PrestaShop/PrestaShop/pull/23687 👍